compile

```diff
@@ -0,0 +1,5 @@
+cd $assets_dir
```

Comment:
Because Gigalixir's default compile script uses brunch, but we're using Phoenix 1.4, we can use a custom static asset compile script here to use webpack:

https://github.com/gjaldon/heroku-buildpack-phoenix-static#compile